app/components/Comments/ModalSource.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Modal from '../Modal/Modal'
+import { popModal } from '../../state/modals/reducer'
+
+import { LoadingFrame } from '../Utils/LoadingFrame'
+import ExternalLinkNewTab from '../Utils/ExternalLinkNewTab'
+
+import styled from 'styled-components'
+import { Flex, Box } from '@rebass/grid'
+import { TimesCircle } from 'styled-icons/fa-solid/TimesCircle'
+import { LinkExternal } from 'styled-icons/octicons/LinkExternal'
+
+const CloseButton = styled(({ size = '1em', ...props }) => (
+  <TimesCircle size={size} cursor="pointer" {...props} />
+))`
+  color: #d0d0d0;
+  &:hover {
+    opacity: 0.75;
+  }
+`
+
+const UrlBarHeader = styled(Flex)`
+  border-bottom: 1px solid #dbdbdb;
+  background-color: white;
+  padding: 10px;
+  position: relative;
+`
+const StyledExternalLinkNewTab = styled(ExternalLinkNewTab)`
+  color: black;
+`
+
+const UrlBarTitle = styled(Box)`
+  display: inline-block;
+  vertical-align: middle;
+  width: 300px;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+`
+
+const IframeContainer = styled.div`
+  background-color: white;
+  height: 100%;
+`
+
+class ModalSource extends React.PureComponent {
+  state = { isLoading: true }
+  renderUrlBarHeader = (url, title, website_name) => {
+    const { popModal } = this.props
+
+    return (
+      <UrlBarHeader justifyContent="space-between">
+        <Box>
+          <StyledExternalLinkNewTab href={url}>
+            <LinkExternal size="1em" />
+            <UrlBarTitle ml="0.2em">{title === null ? website_name : title}</UrlBarTitle>
+          </StyledExternalLinkNewTab>
+        </Box>
+        <CloseButton size="1.5em" title="Close" onClick={popModal} />
+      </UrlBarHeader>
+    )
+  }
+
+  render() {
+    const { source, ...otherProps } = this.props
+    const { url, title, website_name } = source
+    return (
+      <Modal
+        customHeader={this.renderUrlBarHeader(url, title, website_name)}
+        overrideContentStructure
+        isModalSource
+        {...otherProps}
+      >
+        <IframeContainer>
+          {this.state.isLoading && <LoadingFrame />}
+          <iframe
```

Comment:
For security concerns, should we add a [sandbox](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox) attribute to the iframe ?

Reply:
You're right. I'm gonna fix this.